Makefile

```diff
@@ -63,3 +63,21 @@ test-csharp-sdk: typecheck format prepare-aidbox-runme lint
 	cd examples/csharp && dotnet restore
 	cd examples/csharp && dotnet build
 	cd examples/csharp && dotnet test
+
+PYTHON=python3
+PYTHON_SDK_EXAMPLE=./examples/python
+
+test-python-sdk: typecheck format prepare-aidbox-runme lint
+	$(TYPECHECK) --project tsconfig.example-python.json
```

Comment:
Take a look at how `tsconfig.example-python.json` can be moved inside the example path.